xml-parsing/src/test/java/org/sonarsource/analyzer/commons/xml/checks/SonarXmlCheckTest.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Sonar Analyzers XML Parsing Commons
+ * Copyright (C) 2009-2018 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonarsource.analyzer.commons.xml.checks;
+
+import org.assertj.core.util.Lists;
+import org.junit.Test;
+import org.sonarsource.analyzer.commons.xml.XmlFile;
+import org.w3c.dom.Element;
+
+public class SonarXmlCheckTest {
+
+  @Test
+  public void test() throws Exception {
+    TestCheck testCheck = new TestCheck();
+//    SensorContextTester sensorContext = SensorContextTester.create(new File("src/test/resources/"));
```

Comment:
code to delete?

Reply:
I wish commented code would be executed here ... :D